release.sh

```diff
@@ -32,14 +32,18 @@ if ! grep '^globus\-compute\-sdk \& globus\-compute\-endpoint v'"$VERSION"'$' do
 fi
 
 echo "releasing v$VERSION"
-if git tag -s -m "v$VERSION" "$VERSION" ; then
+TAG_STDERR="$(git tag -s -m \"v$VERSION\" \"$VERSION\" 2>&1 > /dev/null)"
+if [[ $? == 0 ]]; then
```

Comment:
FWIW, inside of the subshell, there's not a need to escape the quotes.  Consider:
```shellscript
TAG_STDERR="$(git tag -s -m "v$VERSION" "$VERSION" 2>&1 > /dev/null)"
```

Please double check me, but I think the escapes might even include the quotes in what's given to Git:

```console
$ show_args a \"b\" c
ARG 0: show_args
ARG 1: a
ARG 2: "b"
ARG 3: c

$ SOME_VAR="$(show_args a \"b\" c)"

$ echo -e "$SOME_VAR"
ARG 0: show_args
ARG 1: a
ARG 2: "b"
ARG 3: c
```